runelite-client/src/main/java/net/runelite/client/game/FishingSpot.java

```diff
@@ -25,111 +25,190 @@
 package net.runelite.client.game;
 
 import com.google.common.collect.ImmutableMap;
+import java.util.LinkedHashMap;
 import java.util.Map;
+import java.util.Set;
 import lombok.Getter;
 import net.runelite.api.ItemID;
 import static net.runelite.api.NpcID.*;
+import net.runelite.client.plugins.fishing.FishingTools;
 
+
+@SuppressWarnings("SpellCheckingInspection")
 @Getter
 public enum FishingSpot
 {
-	SHRIMP("Shrimp, Anchovies, Sardine, Herring", "Anchovies", ItemID.RAW_SHRIMPS,
+	SHRIMP("Net/Bait", "Anchovies", new LinkedHashMap<FishingTools, Catches>()
+	{{
```

Comment:
https://github.com/runelite/runelite/runs/7781077871?check_suite_focus=true#step:5:7601

Using double brackets here is clearly not ideal but I'd love to try and elegantly format these maps